src/styles/mailchimp.scss

```diff
@@ -0,0 +1,150 @@
+$mailchimp-color-main: #f2463a;
+$mailchimp-color-light: #f56f65;
+$mailchimp-color-grey: #636363;
+$mailchimp-color-white: #fff;
```

Comment:
can we name the variables in a global scss in the even that we need to use them again? can push to this to sprint 2, lmk